tests/event-processing-tests/build.gradle

```diff
@@ -9,17 +9,18 @@ repositories {
 }
 
 dependencies {
-    implementation 'io.cucumber:cucumber-java:7.2.3'
+    implementation 'io.cucumber:cucumber-java:7.8.1'
```

Comment:
remove build.gradle changes as i think this is from denny's branch. not relevant to this pr